appstore/api/v1/views.py

```diff
@@ -673,6 +689,10 @@ def destroy(self, request, sid=None):
             logger.info("request username: " + str(request.user.username))
             if status.services[0].username == request.user.username:
                 response = tycho.delete({"name": serializer.validated_data["sid"]})
+                # Delete all the tokens the user had associated with that app
+                consumer_id = UserIdentityToken.compute_app_consumer_id(serializer.validated_data["aid"], serializer.validated_data["sid"])
```

Comment:
I think these certainly need more descriptive names than `aid` and `sid`

Reply:
Especially since they get described that way in the logs too

Reply:
None of the other code in this module refers to them beyond aid and sid. It's app id/system id, which are pretty well known identifiers in Tycho I think.

Reply:
Also, the arguments to the function are literally named `app_id, system_id` so it's not like it's ambiguous anyways

Reply:
Oh okay. If they're commonly used names in Tycho then I guess that's fine. I wish they weren't lol but we can refactor that when we refactor Tycho.